biz.aQute.resolve/test/biz/aQute/resolve/ValidatorTest.java

```diff
@@ -64,4 +69,112 @@ public void _testLarger() throws Exception {
 			assertTrue(validator.check());
 		}
 	}
+
+	public void testIndexAgainstLargeTargetPlatform() throws Exception {
+		try (ResolverValidator validator = new ResolverValidator();) {
+			ResourceBuilder resourceBuilder = new ResourceBuilder();
+
+			resourceBuilder.addEE(EE.JavaSE_1_7);
+			resourceBuilder.addManifest(OSGI_CORE.R6_0_0.getManifest());
+
+			// Create a rich base platform by adding more resources.
+
+			File[] files = new File[] {
+					IO.getFile("testdata/liferay-indexes/collaboration-1.0-index.xml"),
+					IO.getFile("testdata/liferay-indexes/forms-and-workflow-1.0-index.xml"),
+					IO.getFile("testdata/liferay-indexes/foundation-1.0-index.xml"),
+					IO.getFile("testdata/liferay-indexes/target.platform.index.xml"),
+			};
+
+			for (File file : files) {
+				try (XMLResourceParser xmlResourceParser = new XMLResourceParser(file.toURI())) {
```

Comment:
The adding of the requirements is irrelevant, you can comment that line and
leave only the capabilities... the error is the same.